src/rsz/test/repair_fanout7.ok

```diff
@@ -11,8 +11,8 @@ load DFF_X1 CK D 300000 metal3 1000
 [INFO ODB-0132]     Created 2 special nets and 306 connections.
 [INFO ODB-0133]     Created 4 nets and 307 connections.
 worst slack -23.63
-[INFO RSZ-0040] Inserted 34 buffers.
-[INFO RSZ-0041] Resized 139 instances.
-[INFO RSZ-0049] Cloned 6 instances.
+[INFO RSZ-0040] Inserted 81 buffers.
+[INFO RSZ-0041] Resized 326 instances.
```

Comment:
Do you understand the jump in these metrics?

Reply:
Different optimization path ... because the cloned gates are now placed differently. And as you can see... the timing has improved (for whatever that is worth since the design is not yet detailed placed).